plugins/scanners/fossid/src/main/kotlin/FossIdNamingProvider.kt

```diff
@@ -51,29 +51,29 @@ class FossIdNamingProvider(
         const val MAX_SCAN_CODE_LEN = 254
     }
 
-    fun createProjectCode(projectName: String): String =
+    fun createProjectCode(repositoryName: String): String =
         namingProjectPattern?.let {
             val builtins = mapOf(
-                "#projectName" to projectName
+                "#repositoryName" to repositoryName
```

Comment:
Should we put the "breaking change" label on the PR ?

I don't remember ORT's policy.

Reply:
This should only be a breaking change if users are required to change their configuration. But my understanding is they don't, or?

Reply:
Yes they do (and I guess that's why @mnonnenmacher  marked is as breaking in the commit message).

One of the builtin naming variable has been renamed. If ORT users are using it, they need to adapt their naming pattern with the new name.
For instance at Bosch we do.

Reply:
> Should we put the "breaking change" label on the PR ?

Didn't we stop doing this when we started publishing release notes?

There will be more breaking changes in a follow-up, I just didn't include them here because I could not yet get the tests to work, but my hope is that I can get them in before the next release.

Reply:
> Didn't we stop doing this when we started publishing release notes?

That's why I was asking, I was unsure.

Reply:
Ah, sorry, I was somehow missing the "label on the PR" part. No, we don't do that anymore.